### PR TITLE
Added simple FAQs and fixed some minor issues

### PR DESCRIPTION
**Pull request by nitrosimon**

### CODE REVIEW

**Review by jans23 — COMMENTED** [review]
The Generic FAQ is generic for Nitrokeys but not applicable for other products.

File: faq/hsm.rst [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+**Q:** Which API can I use?
+    OpenSC: Comprehensive instructions exist for OpenSC framework. There is nitrotool as a more comfortable frontend to OpenSC.
+    GPGSM (from GnuPG): Nitrokey HSM is supported since GnuPG 2.1.
+    Embedded Systems: For systems with minimal memory footprint a read/only PKCS#11 module is provided by the sc-hsm-embedded project.
+    This PKCS#11 module is useful for deployments where key generation at the user's workplace is not required. The PKCS#11 module also supports major electronic signature cards available in the German market.
+    OpenSCDP: The SmartCard-HSM is fully integrated with OpenSCDP, the open smart card development platform. See the public support scripts for details. To import existing keys you can use its SCSH or NitroKeyWrapper.
+
+
+**Q:** How to use the HSM for Login
```
Comment (jans23) [inline faq/hsm.rst:12]: Use proper product name "Nitrokey HSM"

File: faq/shop.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
+Shop and Delivery FAQ
+===========
```
Comment (jans23) [inline faq/shop.rst:2]: Use correct length of heading otherwise Sphinx complains with a warning

File: faq/start.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Start FAQ
```
Comment (jans23) [inline faq/start.rst:1]: Use proper product name

File: nitrophone/index.rst [hunk 4]
```diff
@@ -91,6 +91,10 @@ If you want to check the integrity of the operating system, you can use `remote
 .. figure:: /nitrophone/images/boot-screen.jpg
    :alt: Boot screen
 
+Further informations on GrapheneOs
```
Comment (jans23) [inline nitrophone/index.rst:94]: typo

File: faq/pro.rst [hunk 5]
```diff
@@ -0,0 +1,660 @@
...
+
+P11-glue uses PKCS#11 as glue between crypto libraries and security applications on the open source desktop.
+Articles in German
+
+    Eine allgemeine und gut verständliche Einführung bietet der Artikel Magisches Leuchten im Linux Magazin 2010/12.
+    Eine umfangreiche Anleitung gibt es auf Pro-Linux.
+    Einen ausfuehrlichen Erfahrungsbericht unter CentOS.
+
+
+**Q:** How can I login into Windows?
```
Comment (jans23) [inline faq/pro.rst:57]: I think this and the next section should be instructions instead of FAQ.

**Comment by jans23** [comment]
Why a toplevel `/faq` directory? Isn't it easier for users to have the FAQ within the particular product section?

**Comment by daringer** [comment]
mostly because there is a "generic" FAQ section and because the directory structure reflects the navigation, where the "Frequently Asked Questions" are on the top. But very valid point, at least it should be consistent as nextbox has the faq inside and all other have them in the `faq/` directory. I'll try a compromise...

File: shared-faqs/generic.rst [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+=====================
+
+**Q:** Which Operating Systems are supported?
+   Windows, Linux, and Mac OS X. Also some support (FIDO2) for Android.
+   
+**Q:** What are the default passwords?
+   * **User PIN:** "123456"
+   * **Administrator PIN:** "12345678"
+   * **Firmware Password** (Nitrokey Storage only): "12345678"
+   * **SO-PIN** (Nitrokey HSM only): "3537363231383830"
```
Comment (szszszsz) [inline shared-faqs/generic.rst:11]: That's not true. Nitrokey HSM does not have default PIN, and requires setting it up before the first use.

**Pull request merged**